src/zoid/payment-fields/component.jsx

```diff
@@ -122,6 +122,16 @@ export function getPaymentFieldsComponent() : PaymentFieldsComponent {
                     required: false
                 },
 
+                onContinue: {
+                    type:     'function',
+                    required: false
+                },
+
+                createOrder: {
```

Comment:
Do we need this `createOrder` prop here?

Reply:
No Anshu, we don't need that anymore